tests/plugins/test_fetch_sources.py

```diff
@@ -47,10 +47,10 @@
                         'archives': ['remote-source-first.wrong', 'remote-source-first.tar.gz']},
                        {'name': 'second', 'url': 'second_url',
                         'archives': ['remote-source-second.bad', 'remote-source-second.tar.gz']}]
-RS_TYPEINFO_NO_2 = [{'name': 'first', 'url': 'first_url',
-                     'archives': ['remote-source-first.tar.gz']},
-                    {'name': 'second', 'url': 'second_url',
-                     'archives': ['remote-source-second.tar.gz']}]
+RS_TYPEINFO_NO_TAR_GZ = [{'name': 'first', 'url': 'first_url',
+                         'archives': ['remote-source-first.json']},
+                         {'name': 'second', 'url': 'second_url',
+                         'archives': ['remote-source-second.json']}]
```

Comment:
Would a test case with `.config.json` and `.env.json` be worth it?

Reply:
right, I can add it there to check if they are safely ignored

Reply:
added